packages/demux/src/AloxideActionHandler.ts

```diff
@@ -78,6 +80,20 @@ export class AloxideActionHandler extends AbstractActionHandler {
       .then<void>(() => {});
   }
 
+  private async rollbackIndexState(blockNum: number): Promise<void> {
```

Comment:
Why `blockNum` but not `blockNumber` same as other places?

Reply:
updated